content/people/damiri.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Delaram Amiri
+role: student-phd
+status: Ph.D. Candidate
+year joined: 2015
+portrait: damiri.jpg
+badge: damiri-badge.jpg
+interests:
+    - Healthcare IoT
+    - Sensor Control
+    - Optimization
+    - Edge Computing
+education:
+    - degree: M.S. in Electrical Engineering, Communications
+      institution: Indiana University-Purdue University Indianapolis, U.S.
```

Comment:
Please make sure to replace the minus symbol (`-`) with the Unicode
en-dash symbol (`–`).

Reply:
Copied and pasted this one.
I hope it ok now.
How do you tell the difference? Is there an editor view that makes you see those characters?

Reply:
> Copied and pasted this one. I hope it ok now.

Looks good now. Thanks.

> How do you tell the difference?

I compile the page into HTML and check how it looks on the website.
In the variable-width font used in the resulting page, the difference
is visible.